src/main/java/gmailgdogra/ExcelToExcelParsingWebApp.java

```diff
@@ -13,7 +13,10 @@ public static void main(String[] args) throws IOException {
         SpringApplication.run(ExcelToExcelParsingWebApp.class, args);
         List<SwipeRecord> outputData =
                 PrepareOutputData.from("src/main/resources/testFile.xlsx");
-        outputData.forEach(System.out::println);
+        boolean isSuccess = WriteOutputXlsx.write(outputData);
+        if (isSuccess) {
```

Comment:
While using boolean is ok, explore the modern approach: you use a special class for a result of operation. For example, if the operation succeeds, Result<T> will contain the result of the operation and if it fails, Result<T> contains the reason of the failure (usually an exception). This makes life easier.

Reply:
I'll look into it